fc/parsing/actions.py

```diff
@@ -746,26 +805,50 @@ def resolve_namespaces(self, ns_map):
         for item in itertools.chain(self.inputs, self.outputs, self.optional_decls):
             item.ns_uri = ns_map[item.ns_prefix]
 
-    def associate_model(self, model, units):
-        """Tell this interface what model it is being used to manipulate.
+    def modify_model(self, model, units):
+        """Use the definitions in this interface to transform the provided model.
```

Comment:
Maybe a quick mention or a list of the things that get modified?

Reply:
I might fill that in later, but it's given in the docstrings of the individual methods for what's already implemented, and more will be added as we go.

Reply:
You'd still have to read the code to see that the methods are called... But maybe I'm thinking too much "library" and not enough "application" here...?

Reply:
I've no objection to listing the methods called in order in the docstring. But this will probably change as we add more features, and so will easily get out of sync. I thought it'd be better to document once stabilised.

And yes, I'm treating this more as an application than a library :)

Reply:
I always try and make the documentation say what the _intention_ of the code is (because you can't tell from the code whether it does what it's supposed to do, you can only see what it does).
Could we stick in something like

``The following modifications are made:
- Converting the time units if required (see :meth:...)
- ...
```
with a very obvious `...` to show this documentation isn't finished?

I know we're the only users at present but the whole goal of this project is to make modelling reliable, reproducible, and trustworthy. That means it should be extremely clear to WL users what the WL does, and they should be able to strip away complexities like the web interface and just mess around with FC locally if they are surprised at any of the results. To me the whole project is meaningless if the WL itself isn't easy to open up, inspect, and understand

Reply:
But at the point I wrote this I wasn't sure what the intent was, entirely. It's clearer in #95 so I could probably write the docstring out further there if you like?

Reply:
Yes please!